### DISCUSSION TITLE
When I attempt to view a PDF or other form of a Snapshot, I get a message: Snapshot x exists in DB but resource x does not exist in snapshot dir yet.

### DISCUSSION THREAD

**Discussion by Beck-sys** [ArchiveBox/ArchiveBox#813] [Q&A] [answered]
I followed the directions on doing the archivebox update -t timestamp 
command which was accepted without errors but I still get the same message when I try to view the PDF of the snapshot

**Comment by pirate** [comment, marked as answer]
The PDF extractor is probably failing for some reason on that URL, can you check the ArchiveResult entry for the PDF extractor on that snapshot on the `Log` page in the UI?

There should be some error message output explaining why it failed.

**Reply by Beck-sys** [reply]
Thanks for your reply. I looked at the log (it is doing this for the other URL I added as well). 
From the log, there are no errors. However, it looks like it did not try to save a PDF or image of the web page. 
The log shows: Archive.org, Media, Mercury, readability, WGET, headers, favicon, and title. But nothing else

I thought perhaps it was a problem from where I installed archivebox. I have it on an external SSD with a NTFS file format running on my Linux Mint 32bit laptop. But it does seem to save everything else from the URL just fine. 

Thanks for answering my questions!

**Reply by pirate** [reply]
Based on that output it means you're missing a Chrome/chromium install, can you post the full output of `archivebox --version`.

**Reply by Beck-sys** [reply]
Thanks. I checked the version and the following were marked as disabled: CHROME BINARY, CUSTOM_TEMPLATES_DIR, COOKIES_FILE, CHROME_USER_DATA_DIR, COOKIES_FILE. 

I could reinstall archivebox. I had archivebox installed on the main hard drive of my laptop but then dragged the folder into my SSD drive because I prefer it there. Perhaps it did not copy everything. Is there a command I can use to uninstall archivebox or would deleting the archivebox folder be enough? I'll reinstall it with the APT instructions.

**Reply by pirate** [reply]
No need to reinstall archivebox, you just have to install chrome/chromium. You can do that automatically with `archivebox setup` or manually with `apt install chromium-browser`.

**Reply by Beck-sys** [reply]
Thanks, that worked!!
